js/controllers/window.js

```diff
@@ -6,7 +6,7 @@ function WindowController(editor, settings, analytics, tabs) {
   this.settings_ = settings;
   this.analytics_ = analytics;
   this.tabs_ = tabs;
-  $('#window-close').click(this.close_.bind(this));
+  $('#window-close').click(this.close.bind(this));
```

Comment:
If you have ES6 now, I'd suggestion

`() => {this.close();}`

rather than the bind. It's a lot easier to read, I think, and it still makes it clear that we're passing a callback here.